src/presentation/pages/Home/MyPage/TeamPick/index.tsx

```diff
@@ -1,20 +1,48 @@
-import { useParams } from 'react-router-dom';
-import { useQuery } from 'react-query';
+import { useInfiniteQuery } from 'react-query';
+import { useCallback, useEffect, useState } from 'react';
 
 import { api } from '@api/index';
+import { useScrollHeight } from '@hooks/useScrollHeight';
+import CommonLoader from '@components/common/Loader';
 import CommonNavigation from '@components/common/Navigation';
 import MyPickEmptyView from '@components/common/Empty/MyPick';
+import ProfileListSelectable from '@components/ProfileListSelectable';
 import FeedbackCardList from '@components/FeedbackCard/List';
-import { StMyTeamPick, StMyTeamPickList } from './style';
+import { PICK_PAGE } from '@utils/constant';
+import { StMyTeamPick, StMyTeamList, StMyTeamPickList } from './style';
+import { MyDetail } from '@api/types/user';
 
 function MyTeamPick() {
-  const { userID } = useParams();
-  const { data: feedbackBookmark, isLoading: isTSSBookmarkLoading } = useQuery(
-    ['tssBookmark', userID],
-    () => api.userService.getFeedbackBookmark(userID ?? ''),
-    { useErrorBoundary: true, retry: 1 },
+  const { isBottomReached, isInitialState } = useScrollHeight();
+  const [selectedTeam, setSelectedTeam] = useState<MyDetail | null>(null);
+
+  const fetchFeedbacksByPage = useCallback(
+    async ({ pageParam = 0 }) => {
+      const response = selectedTeam
+        ? await api.userService.getMyFeedbackInfo(selectedTeam.id, pageParam)
+        : await api.userService.getMyFeedbackInfo(pageParam);
```

Comment:
```suggestion
      const response = await api.userService.getMyFeedbackInfo(pageParam, selectedTeam?.id);
```

이 친구도 이렇게 하시면 해결됩니다!

Reply:
정말 너무너무 속이 시원하고 그렇네요 ㅜ 오늘 하루 중에 제일 행복해요 감사합니다 💟